app/views/devise/shared/_links.html.slim

```diff
@@ -0,0 +1,19 @@
+- if controller_name != 'sessions'
+  = link_to "Log in", new_session_path(resource_name)
+  br
+- if devise_mapping.registerable? && controller_name != 'registrations'
+  = link_to "Sign up", new_registration_path(resource_name)
+  br
+- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations'
+  = link_to "Forgot your password?", new_password_path(resource_name)
+  br
+- if devise_mapping.confirmable? && controller_name != 'confirmations'
```

Comment:
you dont have device settings for most of these
either add those or remove for now